### PR TITLE
Migrate TVM and hibernation cases

### PR DESCRIPTION
**Pull request by LiliDeng**
a draft, will push more tvm and hibernation cases, hibernation cases using https://github.com/microsoft/hibernation-setup-tool

### CODE REVIEW

File: lisa/sut_orchestrator/azure/platform_.py [hunk 1]
```diff
@@ -722,6 +724,10 @@ def _get_platform_information(self, environment: Environment) -> Dict[str, str]:
             for key, value in azure_runbook.vm_tags.items():
                 if value:
                     result[key] = value
+        if azure_runbook.security_profile:
+            for key, value in azure_runbook.security_profile.items():
```
Comment (squirrelsc) [inline lisa/sut_orchestrator/azure/platform_.py:728]: Does it need to be retrieved back to test result information?

Comment (LiliDeng) [reply lisa/sut_orchestrator/azure/platform_.py:728]: no

File: microsoft/testsuites/tvm/tvmsuite.py [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+            if cmd_result.exit_code == 0:
+                node.execute(f"rm -rf {repo_file}", sudo=True)
+            echo.write_to_file(
+                content,
+                node.get_pure_path(repo_file),
+                append=True,
+                sudo=True,
+            )
+        elif isinstance(node.os, Debian):
+            content = "\n".join(
```
Comment (squirrelsc) [inline microsoft/testsuites/tvm/tvmsuite.py:57]: Debian has a method `add_repository`. Is it useful to add repos? And refer to it to create one for Fedora and Suse.

File: lisa/sut_orchestrator/azure/platform_.py [hunk 3]
```diff
@@ -1029,6 +1036,8 @@ def _create_deployment_parameters(
         ):
             arm_parameters.use_availability_sets = True
 
+        if self._azure_runbook.enable_hibernation:
```
Comment (squirrelsc) [inline lisa/sut_orchestrator/azure/platform_.py:1039]: Should it be wrapped in a feature?

File: lisa/sut_orchestrator/azure/platform_.py (hunk 1 above)
Comment (squirrelsc) [reply lisa/sut_orchestrator/azure/platform_.py:728]: Well, this part is for retrieve back to information. Will it be removed?

**Comment by squirrelsc** [comment]
@LiliDeng  Does it need anymore?

**Comment by LiliDeng** [comment]
Will create new PR for TVM

**Pull request closed without merging**